docs/src/main/sphinx/admin/graceful-shutdown.rst

```diff
@@ -23,9 +23,12 @@ Keep the following aspects in mind:
 * If you have TLS/HTTPS enabled, you have to ensure the worker certificate is
   CA signed, or trusted by the server calling the shut down endpoint.
   Otherwise, you can make the call ``--insecure``, but that isn't recommended.
-* If :ref:`system information rules <system-file-auth-system_information>` are
-  configured, then the user in the HTTP request must have read and write
-  permissions in the system information rules.
+* The ``default`` :doc:`/security/built-in-system-access-control` does not allow
+  graceful shutdowns. You can use  the ``allow-all`` system access control, or
+  configure :ref:`system information rules
+  <system-file-auth-system_information>` with the ``file`` system access
+  control.
```

Comment:
I think we need to stress the fact it has to be configured on worker nodes. People may expect that configuring access control on coordinator is enough.

Or do you think docs/src/main/sphinx/security/built-in-system-access-control.rst changes are enough?

Reply:
I think ultimately the release notes should provide enough links to go and read more and learn the exact steps necessary from elsewhere in the docs.